cart/application/cartService.go

```diff
@@ -749,12 +815,6 @@ func (cs *CartService) checkProductQtyRestrictions(ctx context.Context, product
 	return nil
 }
 
-func (cs *CartService) publishAddtoCartEvent(ctx context.Context, currentCart cartDomain.Cart, addRequest cartDomain.AddRequest) {
-	if cs.eventPublisher != nil {
-		cs.eventPublisher.PublishAddToCartEvent(ctx, addRequest.MarketplaceCode, addRequest.VariantMarketplaceCode, addRequest.Qty)
-	}
-}
-
```

Comment:
The interface function is now not used anymore within the repo. Maybe we can/should remove it entirely?

Reply:
good point unused unexported functions can safely be removed :)